tox.ini

```diff
@@ -2,16 +2,12 @@
 envlist = unit,integration,quality
 
 [testenv]
-whitelist_externals =
+allowlist_externals =
   make
 commands_pre =
-  pip install -e 'git+https://github.com/openedx/xblock-sdk.git@0.2.2#egg=xblock-sdk==0.2.2'
```

Comment:
This fails in tox for reasons I don't fully understand, but I moved all of the requirements here to the new test_requirements.txt to work around it.